sdk/keyvault/samples/sharelink/ShareLink.csproj

```diff
@@ -34,7 +34,7 @@
     <PackageReference Update="Azure.Core" Version="1.14.0" />
     <PackageReference Update="Azure.Identity" Version="1.4.0" />
     <PackageReference Update="Azure.Security.KeyVault.Secrets" Version="4.1.1" />
-    <PackageReference Update="Microsoft.Azure.AutoRest.CSharp" Version="3.0.0-beta.20210510.2" PrivateAssets="All" />
+    <PackageReference Update="Microsoft.Azure.AutoRest.CSharp" Version="3.0.0-beta.20210602.1" PrivateAssets="All" />
```

Comment:
This package is not available on nuget do we expect customers to reference our tooling feed?

Reply:
It's available in our public feed and I copy the nuget.config from the repo. If we were to make this just a static copy of all the files, that would eliminate that problem as well.

Reply:
I better simulated the ZIP and did find a missing prop to actually make this really work. See my [comment](https://github.com/Azure/azure-sdk-for-net/pull/21598#issuecomment-854996755) below for details.